package.json

```diff
@@ -9,7 +9,12 @@
     "entrypoint": "server/server.js"
   },
   "scripts": {
-    "start": "node $npm_package_config_entrypoint"
+    "start": "node $npm_package_config_entrypoint",
+    "debug": "node --inspect=0.0.0.0:9229 $npm_package_config_entrypoint",	
+    "debug:brk": "node --inspect-brk=0.0.0.0:9229 $npm_package_config_entrypoint",	
+    "debug:legacy": "node --debug=0.0.0.0:5858 $npm_package_config_entrypoint",	
+    "test": "nyc mocha --exit",	
```

Comment:
Since `nyc` and `nodemon` were removed [here](https://github.com/codewind-resources/nodeExpressTemplate/commit/7445b1f99d3a5a68b069ee603e2f7d26f1eab445#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L24) we should not use this script or the next one

Reply:
done